cardano_node_tests/tests/test_pools.py

```diff
@@ -688,6 +690,15 @@ def test_stake_pool_metadata(
         # check `transaction view` command
         tx_view.check_tx_view(cluster_obj=cluster, tx_raw_output=pool_creation_out.tx_raw_output)
 
+        # check dbsync PoolOfflineData table
+        stake_pool_id = pool_creation_out.stake_pool_id
+        pool_params = cluster.g_query.get_pool_params(stake_pool_id=stake_pool_id).pool_params
+
+        # wait a bit for the dbsync thread that fills the PoolOfflineData table
+        time.sleep(60)
```

Comment:
This will sleep even when we are running the test without dbsync. I propose to add
```
if not configuration.HAS_DBSYNC:
```
to the whole code block, and possibly remove it from `check_pool_offline_*` functions.